examples/with-glamorous/pages/index.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react'
-import { css } from 'glamor'
+import { style, rehydrate, css } from 'glamor'
 import glamorous from 'glamorous'
 
+// Adds server generated styles to glamor cache.
```

Comment:
Cool. Out of curiosity what are we rehydrating here? What's stored insides `ids`?